Applications/KNN/jni/main_sample.cpp

```diff
@@ -20,8 +20,6 @@
 #include "tensorflow/contrib/lite/interpreter.h"
 #include "tensorflow/contrib/lite/kernels/register.h"
 #include "tensorflow/contrib/lite/model.h"
-#include "tensorflow/contrib/lite/string.h"
```

Comment:
Minor fix on the sample app. This header is not used and if this header is included, unable to build from tensorflow-lite2